samza-sql/src/main/java/org/apache/samza/sql/interfaces/SqlSystemSourceConfig.java

```diff
@@ -30,9 +30,13 @@
 
 
 /**
- * Configs associated with a system stream.
+ * Configs associated with a system source. Both streams and table sources are supported.
+ * For now, only local tables are supported.
  */
-public class SqlSystemStreamConfig {
+public class SqlSystemSourceConfig {
+
+  private static final String CFG_SAMZA_STREAM_BOOTSTRAP_STREAM = "streams.%s.samza.bootstrap";
```

Comment:
Please refer to the constants that are defined in samza-core/src/main/scala/org/apache/samza/config/StreamConfig.scala, instead of creating new static constant strings here.

Reply:
Sure. I will reuse here.